src/de/dkfz/b080/co/indelcallingworkflow/IndelCallingWorkflow.java

```diff
@@ -37,10 +38,13 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
 
         TumorBamFile bamTumorMerged = new TumorBamFile(_bamTumorMerged);
 
-        if (isControlWorkflow())
+        configurationValues.add(new ConfigurationValue("tumorSample", ((COFileStageSettings) bamTumorMerged.getFileStage()).getSample().getName()));
```

Comment:
There are shorter methods for this in the Workflow and BasicBamFile class.
```suggestion
        setEnv("tumorSample", bamTumorMerged.getSample().getName(), "string")
        setEnv("controlSample", bamControlMerged.getSample().getName(), "string")
```
The values are attached to the context configuration.

Reply:
If you do not like the setEnv syntax, you can also use cvalue(), which does the same.